modules/Users/language/en_us.lang.php

```diff
@@ -214,6 +214,7 @@
     'LBL_MODULE_TITLE' => 'Users: Home',
     'LBL_NAME' => 'Full Name',
     'LBL_SIGNATURE_NAME' => 'Name',
+    'LBL_NAVIGATION_PARADIGM_DESCRIPTION' => 'Select to be able to view modules in the navigation bar based on pre-defined groups. When selected, the "Filter Menu By" feature will appear in the "More" menu.',
```

Comment:
"Filter Menu By" feature? 
So there must be a language string with the value "Filter Menu By" but I can't find it!

Reply:
Actually, I got this string from "SuiteCRM 7.8.20".

Reply:
True... maybe the other one was also deleted. @Dillon-Brown to the rescue!

Reply:
@horus68 Looks like this line was removed accidentally in 7.9.3. Doesn't look like there are any other missing strings related to "Filter Menu By", the string is "Configure Module Menu Filters".